coalib/output/ConsoleInteraction.py

```diff
@@ -548,20 +548,17 @@ def choose_action(console_printer, actions):
     :param actions:         Actions available to the user.
     :return:                Return choice of action of user.
     """
-    console_printer.print(format_lines(
-        "The following actions are applicable to this result:"))
-
     while True:
         console_printer.print(format_lines("*0: " +
-                                           "Apply no further actions."))
+                                           "No further actions."))
```

Comment:
I'm just very picky with those because we found that people don't read the stuff if it's too long and it has a very very big effect on the usability (which is also why I'm super happy this finally happens)